pyvcloud/vcd/org.py

```diff
@@ -373,3 +373,68 @@ def capture_vapp(self,
             rel=RelationType.ADD,
             media_type=EntityType.CAPTURE_VAPP_PARAMS.value,
             contents=contents)
+
+    def create_user(self, user_name, password, role_href, full_name, description, email, telephone, im, alert_email,
```

Comment:
It would be good to have default values, so the client can call the method with only the arguments that wants to set

Reply:
fixed.